payments_service/auth/views.py

```diff
@@ -69,6 +69,7 @@ def post(self, request):
         return Response({
             'buyer_uuid': buyer['uuid'],
             'buyer_pk': buyer['resource_pk'],
+            'buyer_email': email,
```

Comment:
heh. The test changes were mostly refactoring. I was staring at the tests and realized they were doing too much so I split them into smaller tests for each thing tested.